packages/grafana-ui/src/components/Icon/Icon.tsx

```diff
@@ -63,7 +57,11 @@ export const Icon = React.forwardRef<HTMLDivElement, IconProps>(
     return (
```

Comment:
The return logic can be simplified since the only difference between types seems to be an extra class:
```
 <div className={styles.container} {...divElementProps} ref={ref}>
      <Component
        size={svgSize}
        className={cx(
          styles.icon,
          { [styles.orange]: name === 'favorite' && ['default', 'mono'].includes(type) },
          className
        )}
        style={style}
      />
    </div>
```